sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
```

Comment:
maybe I'm missing something here but I thought ```cork create``` (with or without replace) always called ```repo init``` which could retrieve a new manifest. if we get the ```COREOS_SDK_VERSION``` from .repo/manifests we could end up with the wrong version, right?